static/css/components/mybooks-details.less

```diff
@@ -74,7 +74,10 @@
   }
 
   .details-title {
```

Comment:
Here, let's not trim the title, but wrap it instead. See https://github.com/internetarchive/openlibrary/pull/8340/files . We want to display the full title _somewhere_.